.devcontainer/Dockerfile

```diff
@@ -24,7 +24,7 @@ ARG OCAML_VERSION
 USER root
 RUN apt-get install -y m4
 USER opam
-RUN opam update && opam install ocaml-lsp-server
+RUN opam update && opam install ocaml-lsp-server ocamlformat utop
```

Comment:
To še vedno ne doda paketov, ki jih rabi `utop`. Zdi se mi, da bi bila prava rešitev

```
RUN opam update && opam depext --yes --install ocaml-lsp-server ocamlformat utop
```

Poleg tega se mi zdi, da je knjižnjica `m4` zgoraj vključena, ker jo rabi `ocaml-lsp-server`. Če vse skupaj namestimo prek `depext`, vrstica 25 ni več potrebna. Zato tudi ni potreben preklop uporabnika, zato gresta lahko stran tudi vrstici 24 in 26.